doc/source/CHANGELOG.rst

```diff
@@ -7,6 +7,15 @@ The format is largely inspired by keepachangelog_.
 
 .. _0.1.1:
 
+v0.8.0 - Unreleased
+===================
+
+Changed
+-------
+
+- ApproxStringIdMultiMapper no longer expects a string as a key in value,
+  allowing already hashed keys (integers)
```

Comment:
This reads as if it no longer accepted string keys. Maybe rephrase it to make it clearer that you can still use strings.